app/assets/stylesheets/base/_surveyjs.scss

```diff
@@ -15,4 +15,7 @@
   .sv-title {
     font-family: $header-font;
   }
+}
+.sv-question {
+  overflow: visible;
```

Comment:
Could we shift everything from line 11 onwards of app/assets/stylesheets/components/_form.scss into here so it's all in one place? And then I think the remainder of _form.scss (just imports) and _survey.scss can be consolidated into one file?

Reply:
cool. will do that in a separate pr